docs/api/constants.rst

```diff
@@ -741,14 +743,42 @@ Connection
 
       SSL connection is established
 
-.. c:type:: drizzle_socket_owner
+.. c:type:: drizzle_socket_owner_t
 
    Owner of socket connection
 
    .. py:data:: DRIZZLE_SOCKET_OWNER_NATIVE
 
    .. py:data:: DRIZZLE_SOCKET_OWNER_CLIENT
 
+.. c:type:: (deprecated) drizzle_socket_owner
+
+   typedef of :c:type:`drizzle_socket_owner_t`
+
+.. c:type:: drizzle_socket_option_t
```

Comment:
Yeah, just checked. It is in this commit 94cff771a4524c3fc4438487202c3a613eecf355 on v5.x.x